flutter/lib/app_theme.dart

```diff
@@ -1,28 +1,49 @@
+import 'dart:async';
+
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_compose_lambda/app_nav/app_router.dart';
+import 'package:flutter_compose_lambda/pages/blocs/preferences_bloc.dart';
 import 'package:provider/provider.dart';
 
-class ThemeApp extends StatelessWidget {
+class ThemeApp extends StatefulWidget {
```

Comment:
can be kept in `StatelessWidget` so long the `fetchFollowSystemTheme()` can be also init-called in the `ctro` of `PreferencesBloc`.